batch/scripts/_tiff_crosses_dateline.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+#
+# Small Script to check if input raster will
+# cross dateline when converting to EPSG:4326
+#
+# USAGE: ./crosses_dateline.sh infile [outfile]
+#
+# if no outfile is given, the script returns "true" or "false"
```

Comment:
I think with the current modifications, there is no outfile arg, so maybe fix lines 6 and 8?

Reply:
You're right, thanks!